lib/client-assets.php

```diff
@@ -789,12 +789,17 @@ function gutenberg_extend_wp_api_backbone_client() {
 		};
 JS;
 	wp_add_inline_script( 'wp-api', $script );
+	wp_localize_script( 'wp-api', 'wpApiSettings', array(
+		'root'          => esc_url_raw( get_rest_url() ),
+		'nonce'         => ( wp_installing() && ! is_multisite() ) ? '' : wp_create_nonce( 'wp_rest' ),
+		'versionString' => 'wp/v2/',
```

Comment:
Do we ever use it? I think it might be useful, but I don't see any code occurrence inside Gutenberg.

Reply:
This global is the reasons the tests were failing, I just left it as is, it's being used by Core's `wp-api`

Reply:
Okey 👍